### PR TITLE
🐛 Fix "log.SetLogger(...) was never called" in wrok-webhook

### LINKED ISSUE #490: cluster-manager-work-webhook logs can not be displayed
**Describe the bug**
After init a ocm hub, created a manifestWorkReplicaSet. Then i got this error in work-webhook:
```
I0601 04:56:13.522302       1 start.go:70] starting manager
[controller-runtime] log.SetLogger(...) was never called; logs will not be displayed.
Detected at:
	>  goroutine 10904 [running]:
	>  runtime/debug.Stack()
	>  	runtime/debug/stack.go:24 +0x5e
	>  sigs.k8s.io/controller-runtime/pkg/log.eventuallyFulfillRoot()
	>  	sigs.k8s.io/controller-runtime@v0.17.2/pkg/log/log.go:60 +0xcd
	>  sigs.k8s.io/controller-runtime/pkg/log.(*delegatingLogSink).WithName(0xc0004722c0, {0x2a4c271, 0x9})
	>  	sigs.k8s.io/controller-runtime@v0.17.2/pkg/log/deleg.go:147 +0x45
	>  github.com/go-logr/logr.Logger.WithName(...)
	>  	github.com/go-logr/logr@v1.4.1/logr.go:345
	>  sigs.k8s.io/controller-runtime/pkg/webhook/admission.(*Webhook).getLogger.func1()
	>  	sigs.k8s.io/controller-runtime@v0.17.2/pkg/webhook/admission/webhook.go:182 +0x66
	>  sync.(*Once).doSlow(0x0?, 0xc0006240a0?)
	>  	sync/once.go:74 +0xbf
	>  sync.(*Once).Do(...)
	>  	sync/once.go:65
	>  sigs.k8s.io/controller-runtime/pkg/webhook/admission.(*Webhook).getLogger(0xc000578c80, 0xc000270a00?)
	>  	sigs.k8s.io/controller-runtime@v0.17.2/pkg/webhook/admission/webhook.go:180 +0x4d
	>  sigs.k8s.io/controller-runtime/pkg/webhook/admission.(*Webhook).ServeHTTP(0xc000578c80, {0x7f3b0041d520?, 0xc000620780}, 0xc0007dea00)
	>  	sigs.k8s.io/controller-runtime@v0.17.2/pkg/webhook/admission/http.go:117 +0xe4b
	>  sigs.k8s.io/controller-runtime/pkg/webhook/internal/metrics.InstrumentedHook.InstrumentHandlerInFlight.func1({0x7f3b0041d520, 0xc000620780}, 0x2efd300?)
	>  	github.com/prometheus/client_golang@v1.18.0/prometheus/promhttp/instrument_server.go:60 +0xcb
	>  net/http.HandlerFunc.ServeHTTP(0x2efd3f0?, {0x7f3b0041d520?, 0xc000620780?}, 0xc0007bb8a0?)
	>  	net/http/server.go:2136 +0x29
	>  github.com/prometheus/client_golang/prometheus/promhttp.InstrumentHandlerCounter.func1({0x2efd3f0?, 0xc0001a09a0?}, 0xc0007dea00)
	>  	github.com/prometheus/client_golang@v1.18.0/prometheus/promhttp/instrument_server.go:147 +0xb6
	>  net/http.HandlerFunc.ServeHTTP(0x775506?, {0x2efd3f0?, 0xc0001a09a0?}, 0xc00052e580?)
	>  	net/http/server.go:2136 +0x29
	>  github.com/prometheus/client_golang/prometheus/promhttp.InstrumentHandlerDuration.func2({0x2efd3f0, 0xc0001a09a0}, 0xc0007dea00)
	>  	github.com/prometheus/client_golang@v1.18.0/prometheus/promhttp/instrument_server.go:109 +0xc2
	>  net/http.HandlerFunc.ServeHTTP(0x10?, {0x2efd3f0?, 0xc0001a09a0?}, 0xc00052ee2c?)
	>  	net/http/server.go:2136 +0x29
	>  net/http.(*ServeMux).ServeHTTP(0x412885?, {0x2efd3f0, 0xc0001a09a0}, 0xc0007dea00)
	>  	net/http/server.go:2514 +0x142
	>  net/http.serverHandler.ServeHTTP({0x2eede70?}, {0x2efd3f0?, 0xc0001a09a0?}, 0x6?)
	>  	net/http/server.go:2938 +0x8e
	>  net/http.(*conn).serve(0xc0005193b0, {0x2f12cb0, 0xc000b447b0})
	>  	net/http/server.go:2009 +0x5f4
	>  created by net/http.(*Server).Serve in goroutine 185
	>  	net/http/server.go:3086 +0x5cb

```

**To Reproduce**

**Expected behavior**

**Environment ie: OCM version, Kubernetes version and provider:**

**Additional context**

### PR DESCRIPTION
**Pull request by RokibulHasan7**
## Related issue(s)

Fixes #490

### CODE REVIEW

File: pkg/work/webhook/start.go [hunk 1]
```diff
@@ -44,32 +44,34 @@ func (c *Options) RunWebhookServer() error {
 		}),
 	})
 
+	logger := klog.LoggerWithName(klog.FromContext(context.Background()), "work webhook")
```
Comment (zhujian7) [inline pkg/work/webhook/start.go:47]: Consider to move the logger init lines to the beginning of the `RunWebhookServer` method. so it is easy to understand the err check in line 49 is for the `ctrl.NewManager`

Comment (RokibulHasan7) [reply pkg/work/webhook/start.go]: updated

**Pull request merged**